mmv1/third_party/terraform/fwtransport/go/framework_config_test.go

```diff
@@ -14,168 +14,6 @@ import (
 	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
 )
 
-func TestFrameworkProvider_LoadAndValidateFramework_project(t *testing.T) {
```

Comment:
Do you mind adding this function back? Thanks.

Reply:
Added in [f66e3ad](https://github.com/GoogleCloudPlatform/magic-modules/pull/11607/commits/f66e3ad34a8fbe8efcb891364cc0893b1d4e743f)

Reply:
Ah sorry, this is the go rewrite file - I'll push it back